pkg/controller.v1/tensorflow/pod_test.go

```diff
@@ -348,3 +349,160 @@ func TestExitCode(t *testing.T) {
 	}
 	close(stopCh)
 }
+
+func TestScaleDown(t *testing.T) {
```

Comment:
Please add some comments about the behavior for the test case.

Reply:
Will do.